packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -120,7 +120,7 @@ DropdownItem.slotClassNames = {
 DropdownItem.create = createShorthandFactory({ Component: DropdownItem, mappedProp: 'header' })
 
 /**
- * A sub-component of the Dropdown.
- * Used to display items of the dropdown list.
+ * A DropdownItem represents item of Dropdown.
```

Comment:
If the DropdownItem is used to display one option in the list, let's state it like this. As we have lots of Component around the Dropdown - DropdownItem, DropdownLabel etc, so it would be good to make the separation of these components in the description.

Reply:
reformulated :)